components/Capping/styles.scss

```diff
@@ -0,0 +1,20 @@
+//Capping
+$bend:4px;
```

Comment:
Use border radius variables

Reply:
I passed $btn-border-radius-base to $bend @jasoncypret